src/foam/u2/detail/SectionView.js

```diff
@@ -171,7 +171,7 @@ foam.CLASS({
                       return shown ? self.PropertyBorder.create({
                         prop: p,
                         data$: self.data$,
-                        reserveLabelSpace: true
+                        reserveLabelSpace: p.reserveLabelSpace ?? true
```

Comment:
Yes, the `reserveLabelSpace` property doesnt actually exist on Property, it is just something that might be in viewConfig so it would be empty most of the time hence the ?? true